nvflare/fuel/hci/client/file_transfer.py

```diff
@@ -317,6 +323,21 @@ def download_job(self, args, api: AdminAPISpec):
         reply_processor = _DownloadFolderProcessor(self.download_dir)
         return api.server_execute(command, reply_processor)
 
+    def download_job_single_file(self, args, api: AdminAPISpec):
+        if len(args) != 3:
+            return {
+                "status": APIStatus.ERROR_SYNTAX,
+                "details": "usage: download_job_single_file job_id "
+                "file_path\nWhere file_path starts with job or "
+                "workspace.",
+            }
+        job_id = args[1]
+        file = args[2]
```

Comment:
suggest enforce arg to 4
And have something `data_source = args[3]`
data_source need to be either "workspace" or "job"

Reply:
not adding a data_source for now and just keeping job/workspace in the beginning of the path to keep it consistent with what downloads with the download_job command